generators/app/templates/webpack.config.js

```diff
@@ -50,7 +111,7 @@ module.exports = (env, argv) => {
             {
               loader: 'file-loader',
               options: {
-                outputPath: 'assets',
+                OUTPUT_PATH: 'assets',
```

Comment:
🐛 I think this was an inadvertent find and replace.

Reply:
Ahh oops I missed this. Good catch!